cli/commands.go

```diff
@@ -66,6 +66,9 @@ var (
 	ErrSIGTERM           = errors.New("Daemon terminated with SIGTERM")
 )
 
+//The channel gets initialized at boot
+var SignalHandlerChan chan os.Signal
```

Comment:
I understand that SignalHandlerChan in commands needs to Capitalized "C" to be exported. But should I follow the standard camel case on the local SignalHandlerChan varables. for example the variable on line 29 in main.go. Or should I have it the way it is now to be consistent?

Reply:
No, you don't. There is no need to export it from main :)